config/config.go

```diff
@@ -177,6 +177,10 @@ func (data SystemConfigurationCRType) ExternalComputes() ComputesList {
 	var computes ComputesList
 
 	computeNodes := data["spec"].(SystemConfigurationCRType)["externalComputeNodes"]
+	if computeNodes == nil {
```

Comment:
Can we flip that have just one return from the function?

Reply:
done